benchmark/utils/DriverUtils.swift

```diff
@@ -107,9 +107,12 @@ struct TestConfig {
   /// The filters applied to our test names.
   var filters = [String]()
 
-  /// The tag that we want to run
+  /// The tags that we want to run
```

Comment:
Btw, one other thing that we could do to make this code even better is to use an OptionSet instead of a plain Set. I just forgot to make this change.

Reply:
Yeah. but for that the `BenchmarkCategory` would have to become backed by an `Int`, and I think there is some reliance on them being `String`s.